game/songparser-sm.cc

```diff
@@ -247,12 +247,12 @@ Notes SongParser::smParseNotes(std::string line) {
 			else if(notetype >= 'a' && notetype <= 'z') note.type = Note::Type::TAP;
 			else if(notetype >= 'A' && notetype <= 'Z') note.type = Note::Type::TAP;
 			else throw std::runtime_error(std::string("Invalid note command: ") + notetype);
-			note.note = i;
+			note.note = static_cast<float>(i);
 			chord[i] = note;
 		}
 		chords.push_back(chord);
 	}
-	m_tsEnd = std::max(m_tsEnd, measure * 16);
+	m_tsEnd = std::max(m_tsEnd, static_cast<unsigned>(std::round(measure * 16.0)));
```

Comment:
not sure round is useful as casting a float to an int is already some kind of round (ok, actually it truncates...)